docs/docker/building-net-docker-images.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Building .NET Core Docker Images
+description: Understanding Docker images and .NET Core
+keywords: .NET, .NET Core, Docker
+author: spboyer
+manager: wpickett
+ms.date: 08/29/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 03c28597-7e73-46d6-a9c3-f9cb55642739
+---
+
+
+#Building Docker Images for .NET Core Applications
+
+In order to get an understanding of how to use .NET Core and Docker together, we must first get to know the different Docker images that are offered and when is the right use case for them. Here we will walk through the variations offered, build an ASP.NET Core Web API, use the Yeoman Docker tools to create a debuggable container as well as peek at how Visual Studio Code can assist in the process. 
+
+## Docker Image Optimizations
+
+When building docker images for developers, we focused on 3 main scenarios
```

Comment:
There are a couple of instances where docker is lower caps. I think the right branding name is with a capital D.
Also, according to Microsoft style guide, you should "spell out whole numbers from zero through nine"